test/index.test.js

```diff
@@ -17,33 +20,26 @@ describe('electron-installer-dmg', () => {
   describe('with app', () => {
     const appPath = path.resolve(__dirname, 'fixture');
 
-    before(function downloadElectron(done) {
+    before(async () => {
       this.timeout(120000);
 
-      electronDownload({
-        version: '2.0.4',
-      }, (downloadErr, zipPath) => {
-        if (downloadErr) return done(downloadErr);
-        zip.unzip(zipPath, appPath, done);
-      });
+      const zipPath = await download('2.0.4');
+      await unzip(zipPath, appPath);
     });
 
-    it('should succeed in creating a DMG', function testCreate() {
+    it('should succeed in creating a DMG', async () => {
       this.timeout(30000);
       const createDMG = require('../'); // eslint-disable-line global-require
       const dmgPath = path.resolve(__dirname, 'fixture.dmg');
-      return createDMG.p({
+      await createDMG({
         appPath,
         dmgPath,
         name: 'Test App',
-      }).then(() => {
-        assert(fs.existsSync(dmgPath), 'dmg should exist');
       });
+      assert(await fs.pathExists(dmgPath), 'dmg should exist');
     });
 
-    afterEach(() => {
-      fs.unlinkSync(`${appPath}.dmg`);
-    });
+    afterEach(async () => fs.unlink(`${appPath}.dmg`));
```

Comment:
Doesn't need to be `async`

Reply:
I'd rather be explicit than implicit, but point taken.